patches/uClibc-ng/1.0.12/100-provide-_obstack_free.patch

```diff
@@ -0,0 +1,12 @@
+diff -ur uClibc-ng-1.0.12.orig/libc/misc/gnu/obstack.c uClibc-ng-1.0.12/libc/misc/gnu/obstack.c
+--- uClibc-ng-1.0.12.orig/libc/misc/gnu/obstack.c	2016-03-13 15:08:44.408962824 -0700
++++ uClibc-ng-1.0.12/libc/misc/gnu/obstack.c	2016-03-13 15:13:30.129322998 -0700
+@@ -385,7 +385,7 @@
+     abort ();
+ }
+ 
+-# if 0
++# if 1
+ /* Older versions of libc used a function _obstack_free intended to be
+    called by non-GCC compilers.  */
+ strong_alias (obstack_free, _obstack_free)
```

Comment:
Forgot the most important part: I am not subscribed to uclibc-ng mailing list and attempt at subscription did not generate any email confirmation. Thus, the email with the patch I sent is awaiting moderator's approval.